src/inodes/INodeManager.ts

```diff
@@ -289,17 +273,15 @@ class INodeManager {
     if (typeof params[0] !== 'number') {
       navigated = params.shift() as Readonly<{ dir: INodeIndex; name: string }>;
     }
-    const lockRequests = (params as Array<INodeIndex>).map<
-      [INodeIndex, typeof Lock]
-    >((ino) => [ino, Lock]);
+    const lockRequests = (params as Array<INodeIndex>).map((iNode) =>
+      iNode.toString(),
```

Comment:
Is this necessary? I mean I forgot whether the existing locking system in DB already converts to string naturally.

Reply:
It should, but i encountered a problem with it. I'll check it again.

Reply:
Trying to do a `tran.lock(...inos)` gives us the type error `TS2345: Argument of type 'INodeIndex' is not assignable to parameter of type 'string | [key: ToString, type?: "read" | "write" | undefined, timeout?: number | undefined]'.   Type 'INodeIndex' is not assignable to type '[key: ToString, type?: "read" | "write" | undefined, timeout?: number | undefined]'.`

Using `await tran!.lock([inos[0], 'write'])` works but we'd have to map it to the `[key, 'write']` form anyway. seems like just using a key won't auto covert to a string. Seems like the API needs to be updated tp `ToString | string | ect`?